tests/Common/TokensTest.php

```diff
@@ -331,7 +331,7 @@ public function testTokenRefreshWhenTokenIsString(): void
     public function testTokenGetWhenTokenIsString(): void
     {
         $this->expectException(ClientException::class);
-        $this->expectExceptionMessage('Argument "id" is expected to be type "int", value "foo" given.');
+        $this->expectExceptionMessage('resource not found');
```

Comment:
Toto byla sračka prakticky spíš internal než aby to šlo na klienta špatný typehint akorát sa tatkto v zendu přes argument resolver propisuje.